### ISSUE TITLE
Behaviour of anti-forgery token has changed 7.14.0 -> 7.15.0

### ISSUE THREAD

**Issue by simonhawes** [umbraco/Umbraco-CMS#5850]
Previously working code when submitting forms via ajax...

            if (httpContext.Request.IsAjaxRequest() == true)
            {
                // check if the RequestVerificationToken header exists
                additionalToken = httpContext.Request.Headers["RequestVerificationToken"];
            }
            else
            {
                additionalToken = httpContext.Request.Form["__RequestVerificationToken"];
            }
            AntiForgery.Validate(cookieToken, additionalToken);

Is now failing under 7.15.0  In fact the token from the httpContext.Request.Form is valid even when submitting the form via AJAX, hence the @Html.AntiForgeryToken() tag is producing an incorrect token.

**Comment by Shazwazza** [comment]
Can you provide some more context and steps to replicate please?

**Comment by simonhawes** [comment]
Code was using an anti-forgery token generated in the _layout.cshtml when submitting forms via ajax.  We had multiple forms on a single page so the same token was used for all forms.  This was incorrect but worked in 7.14.0.    When we upgraded to 7.15.0 this no longer worked.  I have updated our code to use the specific token from within each form and the code is now implemented correctly and works as it should.
